src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -1195,7 +1201,59 @@ private:
     auto req = apiCap.restoreRequest();
     req.setToken(percentDecode(buf));
 
-    return req.send().getCap().castAs<Identity>();
+    // Don't directly call `getCap()` on the `RemotePromise` because we want to make sure
```

Comment:
Ah, I hadn't realized that I could both pipeline and detect errors like that. Updated.